Dockerfile

```diff
@@ -0,0 +1,29 @@
+ARG base_image=ghcr.io/alphagov/govuk-ruby-base:2.7.6
+ARG builder_image=ghcr.io/alphagov/govuk-ruby-builder:2.7.6
+ 
+FROM $builder_image AS builder
+
+WORKDIR /app
+
+COPY Gemfile* .ruby-version /app/
+
+RUN apt update && apt install libpq-dev -y && \
+    bundle install
```

Comment:
Couple of minor improvements:
* use `--no-install-recommends` so as not to install unnecessary packages (this is mainly to reduce noise from vulnerability monitoring tools, but also helps with image size of course)
* run the `bundle install` separately from the package installation, to take better advantage of layer cachine.
```suggestion
RUN apt update && apt install -y --no-install-recommends libpq-dev
RUN bundle install
```

Normally we'd also do `&& rm -rf /var/lib/apt/lists/*` as part of the `RUN apt-get` instruction, but since this is the builder image it's fine not to. (https://docs.docker.com/develop/develop-images/dockerfile_best-practices/#apt-get)

Reply:
Actually I just added libpq-dev to the builder image — much cleaner that way. https://github.com/alphagov/govuk-ruby-images/pull/11

So this can just become `RUN bundle install` here.